session_items.py

```diff
@@ -62,8 +62,16 @@ def save_item(item):
         item: The item to save.
     """
     existing_items = get_items()
+    print(existing_items)
```

Comment:
Print statements during development are fine, but should normally be removed before merging anything to master (pprint statements too).